### PR TITLE
Databases week2

### PR DESCRIPTION
**Pull request by andrii1**

### CODE REVIEW

File: databases/week2/part1.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+/*Change the title of a task*/
+UPDATE task SET title = "Wash pants" WHERE id = 36
+/*Change a task due date*/
+UPDATE task SET due_date = '2022-10-29 06:55:16' WHERE id = 36
+/*Change a task status*/
+UPDATE task SET status_id = '2' WHERE id = 36
+/*Mark a task as complete*/
+UPDATE task SET status_id = '3' WHERE id = 36
+/*Delete a task*/
+DELETE FROM task WHERE id = 36
```
Comment (27ManishaPatel) [inline databases/week2/part1.sql:13]: You can use formatting  tools to formate the documents 
just a suggestion!

File: databases/week2/school-database.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE school
+    DEFAULT CHARACTER SET = 'utf8mb4';
```
Comment (27ManishaPatel) [inline databases/week2/school-database.sql]: I am not able to see any task in your school database 🤔

File: databases/week2/creating-database.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `consultations` (
+  `doctor_id` int(10) unsigned NOT NULL,
+  `patient_id` int(10) unsigned NOT NULL,
+  `occurs_on` DATETIME NOT NULL,
+  PRIMARY KEY(`doctor_id`, `patient_id`),
+  CONSTRAINT `fk_consultations_doctor` FOREIGN KEY (`doctor_id`) REFERENCES `doctors` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `fk_consultations_patient` FOREIGN KEY (`patient_id`) REFERENCES `patients` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```
Comment (27ManishaPatel) [inline databases/week2/creating-database.sql:23]: Good database! and you can insert som e values in it as its optional 😊
well done !

**Review by Elia-Git — CHANGES_REQUESTED** [review]
Pls implement feedbacks from Manisha.

File: databases/week2/school-database.sql (hunk 2 above)
Comment (andrii1) [reply databases/week2/school-database.sql]: hm weird, added again, maybe I didn't save it as I remember doing it

**Pull request merged**